src/Test.UnitTests.Sarif/TestData/SarifVersionOneToCurrentVisitor/Inputs/WebScout.sarif

```diff
@@ -0,0 +1,577 @@
+{
+    "$schema": "http://json.schemastore.org/sarif-1.0.0",
+    "version": "1.0.0",
+    "runs": [{
+        "tool": {
+            "name": "WebScout"
+        },
+        "invocation": {
+            "startTime": "2019-05-02T06:47:18.794Z",
+            "endTime": "2019-05-02T07:02:37.638Z",
+            "properties": {
+                "scanConfigurationId": 128483,
+                "rootUrls": [
+                    "https://example.com/"
+                ]
+            }
+        },
+      "results": [
+        {
+          "ruleId": "WS099",
+          "level": "error",
```

Comment:
> "level": "error", [](start = 9, length = 18)

Question: ok, i manually edited this before uploading the file. The version which was sent to us had this value as "5". This failed for me when i plugged it as a unit test case. I just want to understand how this works for our customers? where does the enum conversion happen?